File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/MonthCalendar/Programming/SettingValuePage.vb
Namespace CompanionKit.Controls.MonthCalendar.Programming

	Public Class SettingValuePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles Click event of the Button that set date for the select MonthCalendar
        ''' </summary>
        Private Sub mobjSetDateButton_Click(sender As Object, e As EventArgs) Handles mobjSetDateButton.Click
            If Me.mobjEnableMonthCalendarRadioButton.Checked Then
                ' Set date of the DateTimePicker for the enable MonthCalendar
                Me.mobjEnableMonthCalendar.Value = Me.mobjSelectDateDateTimePicker.Value
            Else
                ' Set date of the DateTimePicker for the disable MonthCalendar
                Me.mobjDisableMonthCalendar.Value = Me.mobjSelectDateDateTimePicker.Value
            End If
        End Sub

        ''' <summary>
        ''' Handles Load event of the example' UserControl.
        ''' </summary>
        Private Sub SettingValuePage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            ' Set checked state of the Radiobutton that indicates whether selected date will be set for enable MonthCalendar
            Me.mobjEnableMonthCalendarRadioButton.Checked = True
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/MonthCalendar/Programming/SettingValuePage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.MonthCalendar.Programming
{
    public partial class SettingValuePage : UserControl
    {
        public SettingValuePage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Click event of the Button that set date for the select MonthCalendar
        /// </summary>
        private void mobjSetDateButton_Click(object sender, EventArgs e)
        {
            if (this.mobjEnableMonthCalendarRadioButton.Checked)
            {
                // Set date of the DateTimePicker for the enable MonthCalendar
                this.mobjEnableMonthCalendar.Value = this.mobjSelectDateDateTimePicker.Value;
            }
            else
            {
                // Set date of the DateTimePicker for the disable MonthCalendar
                this.mobjDisableMonthCalendar.Value = this.mobjSelectDateDateTimePicker.Value;
            }
        }
    }
}